src/mpi/comm/comm_split_type.c

```diff
@@ -607,6 +607,92 @@ static int network_split_by_minsize(MPIR_Comm * comm_ptr, int key, int subcomm_m
   fn_fail:
     goto fn_exit;
 }
+
+static int network_split_by_min_memsize(MPIR_Comm * comm_ptr, int key, long min_mem_size,
+                                        MPIR_Comm ** newcomm_ptr)
```

Comment:
Why isn't `min_memsize` basically just `minsize`, but by calculating the number of nodes based on the available memory?

Reply:
I think it's reasonable to assume that.  I don't know of any reasonable system where that's not true, and I'm sure we'll run into a number of other weird problems if that weren't true.